CHANGELOG.md

```diff
@@ -3,6 +3,7 @@ Changelog
 
 master (unreleased)
 -------------------
+* Add support for `{% set %}` block assignments as in jinja2.
```

Comment:
Can you add a link to the PR here, like the other changelist entries? Thanks! (You can also add a thanks to yourself, if that doesn't feel too weird! Think of it as the project thanking you, you're just doing the data entry :-) )

Also, blank line between header and first list element.